mobile/src/main/java/rs/readahead/washington/mobile/views/activity/CollectFormEntryActivity.java

```diff
@@ -91,9 +92,9 @@ public class CollectFormEntryActivity extends MetadataActivity implements
     @BindView(R.id.screenFormView)
     ViewGroup screenFormView;
     @BindView(R.id.prevSection)
-    AppCompatButton prevSectionButton;
+    TextView prevSectionButton;
```

Comment:
maybe we should refactor to `ViewBinding` whenever we have the chance, it's already enabled in the project.